.github/workflows/tox.yml

```
@@ -179,6 +179,11 @@ jobs:
       with:
         name: logs.zip
         path: .tox/**/log/
+    - name: Report junit failures
+      uses: shyim/junit-report-annotations-action@master
+      with:
+        path: .tox/junit*.xml
```

this is more precise
```suggestion
        path: .tox/junit.*.xml
```